Jetcaster/wear/src/main/java/com/example/jetcaster/podcasts/PodcastsScreen.kt

```diff
@@ -135,40 +93,114 @@ fun PodcastsScreen(
         modifier = modifier
     ) {
         when (podcastsScreenState) {
-            is PodcastsScreenState.Loaded -> {
-                EntityScreen(
-                    columnState = columnState,
-                    headerContent = {
-                        ResponsiveListHeader(
-                            contentPadding = ListHeaderDefaults.firstItemPadding()
-                        ) {
-                            Text(text = stringResource(id = R.string.podcasts))
-                        }
-                    },
-                    content = {
-                        items(count = podcastsScreenState.podcastList.size) {
-                                index ->
-                            MediaContent(
-                                podcast = podcastsScreenState.podcastList[index],
-                                downloadItemArtworkPlaceholder = rememberVectorPainter(
-                                    image = Icons.Default.MusicNote,
-                                    tintColor = Color.Blue,
-                                ),
-                                onPodcastsItemClick = onPodcastsItemClick
-
-                            )
-                        }
-                    }
+            is PodcastsScreenState.Loaded -> PodcastScreenLoaded(
+                columnState = columnState,
+                podcastList = podcastsScreenState.podcastList,
+                onPodcastsItemClick = onPodcastsItemClick
+            )
+            PodcastsScreenState.Empty ->
+                PodcastScreenEmpty(onDismiss)
+            PodcastsScreenState.Loading ->
+                PodcastScreenLoading(columnState)
+        }
+    }
+}
+
+@Composable
+fun PodcastScreenLoaded(
```

Comment:
Nit: Make sure to add modifiers to these composables. Same in `QueueScreen`.

Reply:
Is it worth adding lint to fail on these?

Reply:
Actually maybe it won't for this, just based on order?

Reply:
> Is it worth adding lint to fail on these?

Error might be too aggressive—it should warn at the very least.

Also, if you are using live templates, typing "comp" in the editor will automatically create the following template for you so it's easy to add a composable with a default modifier (I know this was in AS preview and I think should be available now in the stable channel with jellyfish):

```
@Composable
fun <cursor here>(modifier: Modifier = Modifier) {
}
```

Reply:
thanks, added